code/game/objects/items/pet_carrier.dm

```diff
@@ -89,16 +92,19 @@
 	if(target.mob_size > max_occupant_weight)
 		if(ishuman(target))
 			var/mob/living/carbon/human/H = target
```

Comment:
This line is unnecessary, resting is a living var and the iscatperson() define, checks if the mob is human before hand.
So no phantom issues with the above suggestion, I believe.